src/migrateTable.ts

```diff
@@ -65,13 +69,24 @@ export function migrateTable(
 
   const legacyColumns =
     legacyTableState.value?.body?.configuration?.tabular?.columns;
-  const columnWidths = legacyColumns
-    ? _migrateTableColumnWidths({
-        legacyColumns,
-        mapping,
-        cube,
-      })
-    : {};
+
+  const columnsAxis = query.mdx?.axes[0];
+  const columnLevels = columnsAxis ? getLevels(columnsAxis, { cube }) : [];
+
+  const maxLevelDepth = _max(
```

Comment:
We need a sum here, not a max. Indeed, if the table has Desk (3rd level in its hierarchy) and Counterparty (2nd level in its hierarchy) on rows, then a row in the tree column will typically contain 5 elements and therefore the depth is 5:

| LegalEntityA | BusinessUnitA | DeskA | Asahi | Asahi Chemical Ind. |
|---------------|-----------------|--------|--------|-----------------------|